docs/additional-functionality/rapids-shuffle.md

```diff
@@ -100,50 +145,16 @@ system if you have RDMA capable hardware.
 Within the Docker container we need to install UCX and its requirements. These are Dockerfile
 examples for Ubuntu 18.04:
 
-##### Without RDMA:   
-The following is an example of a Docker container with UCX 1.10.1 and cuda-11.0 support, built
-for a setup without RDMA capable hardware:
-
-```
-ARG CUDA_VER=11.0
-
-# Now start the main container
-FROM nvidia/cuda:${CUDA_VER}-devel-ubuntu18.04
-
-RUN apt update
-RUN apt-get install -y wget 
-RUN cd /tmp && wget https://github.com/openucx/ucx/releases/download/v1.10.1/ucx-v1.10.1-ubuntu18.04-mofed5.x-cuda11.0.deb
-RUN apt install -y /tmp/*.deb && rm -rf /tmp/*.deb
-```
-
-##### With RDMA:
-The following is an example of a Docker container that shows how to install `rdma-core` and 
-UCX 1.10.1 with `cuda-11.0` support. You can use this as a base layer for containers that your 
-executors will use.
+The following are examples of Docker containers with UCX 1.10.1 and cuda-11.0 support. The 
+examples are divided by OS type and RDMA-capable hardware availability.
```

Comment:
Now that it's in table form, the table headers makes the second sentence redundant.
```suggestion
The following are examples of Docker containers with UCX 1.10.1 and cuda-11.0 support.
```

Reply:
Thanks good point. I've added this one, which unfortunately dismissed your approval.